python/cudf/cudf/testing/dataset_generator.py

```diff
@@ -388,7 +388,12 @@ def rand_dataframe(
                     ColumnParameters(
                         cardinality=cardinality,
                         null_frequency=null_frequency,
-                        generator=int_generator(dtype=dtype, size=cardinality),
+                        generator=int_generator(
+                            dtype=dtype,
+                            size=cardinality,
+                            min_bound=meta.get("min_bound", None),
```

Comment:
Would it be better to put the default values inline here instead of below?

Reply:
We are going to use those `int_generator`, `float_generator` call's elsewhere in list generation and struct generation too. So inlining them here will force us to inline them elsewhere too.